### DISCUSSION TITLE
Copilot Chat in Visual STUIDO 2022...

### DISCUSSION THREAD

**Discussion by stephenjstraley** [community/community#57269] [Copilot Conversations]
### Select Topic Area

Question

### Body

I received the email that "in the GetHub Copilot chat private beta".

Copilot is working in the IDE but the "Github Copilot does not appear on my VIEW menu AND there is no "Ask Copilot" at the top of a right-mouse click menu in the code window.

In addition, the email I received said "For Visual Studio, please use this survey to raise any bugs and issues you find... and when I click on "use this survey" link I get the LOVELY message that the form has been retired.

PLEASE... I want to use the Copilot Chat option within Visual Studio 2022... HOW!?!?

**Comment by stephenjstraley** [comment]
I had to UNinstall the plugin in VS2022 and re-install - now it appears.

**Reply by ker2x** [reply]
This worked for me. thx :)
